docs/docs/providers/linkedin.md

```diff
@@ -31,7 +31,12 @@ import LinkedInProvider from "next-auth/providers/linkedin";
 providers: [
   LinkedInProvider({
     clientId: process.env.LINKEDIN_CLIENT_ID,
-    clientSecret: process.env.LINKEDIN_CLIENT_SECRET
+    clientSecret: process.env.LINKEDIN_CLIENT_SECRET,
```

Comment:
in `packages/core`, we already change the linkedin provider to type OIDC which essentially has the same effect with this change. We won't merge it to v4, apologize for inconvenience - you can still pass these options to the provider in your code.